src/core/src/consensus_state.rs

```diff
@@ -169,9 +168,9 @@ impl ConsensusState{
     }
 
     pub fn validator_did_claim_block(&mut self, validator_info: &ValidatorInfo, wait_certificate: &WaitCertificate, poet_settings_view: &PoetSettingsView ) -> (){
-      self.aggregate_local_mean += wait_certificate.local_mean;
+      self.aggregate_local_mean += 5.5_f64; //wait_certificate.local_mean;
       self.total_block_claim_count += 1;
-      self.population_samples.push_back( PopulationSample{ wait_time: wait_certificate.wait_time , local_mean: wait_certificate.local_mean}); 
+      self.population_samples.push_back( PopulationSample{ wait_time: wait_certificate.wait_time , local_mean: 5.5_f64}); //wait_certificate.local_mean}); 
```

Comment:
Here also

Reply:
whether local_mean will be in wait_cert or not is still under discussion. Keeping the placeholders in comment to indicate  that.